client/state/selectors/get-recommended-plugins.js

```diff
@@ -0,0 +1,16 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns a list of recommended plugins for the given site.
+ *
+ * @param  {Object}    state 		Global state tree
+ * @param  {Number}    siteId 	The ID of the site we're querying
```

Comment:
Nit: it's common and makes it more readable to vertically align the different fragments of the jsdoc.